chrome/content/zotero/itemTreeColumns.jsx

```diff
@@ -323,18 +324,71 @@ const COLUMNS = [
 	}
 ];
 
+
+
+/**
+ * @type {ItemTreeColumnOption[]}
+ * @constant
+ */
+const CUSTOM_COLUMNS = [];
+
+/**
+ * 
+ * @param {string} dataKey 
+ * @returns {ItemTreeColumnOption | {}}
+ */
 function getDefaultColumnByDataKey(dataKey) {
-	return Object.assign({}, COLUMNS.find(col => col.dataKey == dataKey), {hidden: false});
+	return Object.assign({}, getColumns().find(col => col.dataKey == dataKey), { hidden: false });
 }
 
+/**
+ * 
+ * @param {string[]} dataKeys 
+ * @returns {ItemTreeColumnOption[]}
+ */
 function getDefaultColumnsByDataKeys(dataKeys) {
-	return COLUMNS.filter(column => dataKeys.includes(column.dataKey)).map(column => Object.assign({}, column, {hidden: false}));
+	return getColumns().filter(column => dataKeys.includes(column.dataKey)).map(column => Object.assign({}, column, { hidden: false }));
+}
```

Comment:
The above functions no longer only get default columns (since `getColumns()` includes custom columns). We should evaluate if we still need a function that only retrieves default columns and either add `getColumnsByDataKeys()` functions, or rename these. Or perhaps change this function to exclude custom columns with a boolean parameter after renaming it.

Reply:
Yes, indeed. Or we just give a parameter like a filter, call it like `getColumns([{dataKey: "title", custom: false}])` and it returns all matches.

Reply:
Sure if that's not too complicated.

Reply:
Done